docs/cpp/config-clang-mac.md

```diff
@@ -46,7 +46,7 @@ cd helloworld
 code .
 ```
 
-By starting VS Code in a folder, that folder becomes your *workspace*. VS Code stores user settings that are specific to that workspace in `.vscode/settings.json`. In this tutorial, we'll add three additional files to the `.vscode` folder:
+The **code .** command opens VS Code for Mac in the current working directory, which becomes your *workspace*. Our task is to add three files to the workspace that will tell VS Code how to compile and debug our program. VSCode will place these files in a `.vscode` subdirectory that it will create for us:
```

Comment:
Italics doesn't look great on code.visualstudio.com so it is better to use bold for term emphasis. **workspace**
Also please use either "Visual Studio Code" (first mention) or "VS Code" (everywhere else in the article). Don't use "VSCode".
There isn't a product called "VS Code for Mac" or "VS Code for Windows", so just use "VS Code".
This comment applies to all four tutorials.